### ISSUE TITLE
Consider keeping pixel-copying versions of ops in TransformNamespace

### ISSUE THREAD

**Issue by stelfrich** [imagej/imagej-ops#434]
@ctrueden in a discussion about #433:

> We can debate whether to keep a pixel-copying version as `transform.scale` still.

Most of the ops in the `TransformNamespace` are nowadays wrappers for `Views.*`. `Ops.Transform.Scale` implemented a scaling operation that copied pixel-wise. With chaining it is possible to get a copy of the output of any `Views`-based transform anyway. But should we consider exposing convenience methods for `Input > Transform > Copy = Output`?

**Comment by ctrueden** [comment]
The general idea is for the `burnIn` op to solve this: you can call `burnIn` and it will "copy as needed". The `View` marker interface, and its proper propagation, is a necessity for this to work well.

**Comment by stelfrich** [comment]
Thanks for the heads up @ctrueden. I wasn't really aware of the gory details of the topic (btw thanks for the [great recap](https://github.com/imagej/imagej-ops/issues/173#issuecomment-214889454)).

> The `View` marker interface, and its proper propagation, is a necessity for this to work well.

You are talking about the output of an op implementing the `View` interface, right?

**Comment by ctrueden** [comment]
> You are talking about the output of an op implementing the View interface, right?

Correct. Since your new `scaleView` op returns the result of a `Views` method call, this should already be the case, fortunately.
